plugins/modules/xen_orchestra.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: xen_orchestra
```

Comment:
Maybe this module should be called `xen_orchestra_instance` if it's about managing Xen Orchestra instances.

Reply:
Should I rename the module (`xen_orchestra.py` -> `xen_orchestra_instance.py`) altogether as well?

Reply:
Yes, the module's filename must equal the `name` in its documentation.

Reply:
Done